product.rb

```diff
@@ -0,0 +1,9 @@
+$LOAD_PATH << '.'
+require 'section'
+class Product < Section
+  def something
+  end
+end
+
+prod = Product.new
```

Comment:
Higher level method at the company level to show department employees. 

eg: 
# class Company 
def employee_details
  
end

# sectionType can be "Product" or "Service"
```
def employee_details_for(sectionType)
  if section == 'Product' 
    section = Product.new
  elsif section == 'Service'
    section = Service.new
  end
  section.employee_details
end
```
Or using met programming
```
def employee_details_for(sectionType)
  section =  'section'.eval.class.new 
  section.employee_details
end
```